spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -2289,3 +2289,9 @@ hello world 1458595078733
 hello world 1458595079734
 hello world 1458595080735
 ----
+
+=== Deploying Stream applications on CloudFoundry
+
+On CloudFoundry services are usually exposed via a special environment variable called https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES[VCAP_SERVICES].
+
+When configuring your binder connections, you can use the values from an environment variable as explained on the http://docs.spring.io/spring-cloud-dataflow-server-cloudfoundry/docs/current-SNAPSHOT/reference/htmlsingle/#getting-started-ups[dataflow cloudfoundry server] docs.
```

Comment:
The intention was to show how to parse a VCAP_SERVICES variable. It came as a ticket opened on the kafka-binder because the user had no idea that he could use ${} to evaluate the VCAP env var. I just wanted a few samples on how to use it, and the kafka demo with cups seems to have a fair coverage of the usage of $VCAP env